charts/fleet/templates/deployment.yaml

```
@@ -64,6 +64,18 @@ spec:
         - name: CATTLE_DEV_MODE
           value: "true"
         {{- end }}
+        {{- if $.Values.controller.reconciler.workers.gitrepo }}
+        - name: CONTROLLER_GITREPO_RECONCILER_WORKERS
```

```suggestion
        - name: GITREPO_RECONCILER_WORKERS
```

I'd remove the controller prefix on all envs, for brevity.

---

Just changed this TY Mario !